sw/device/lib/testing/pinmux_testutils.c

```diff
@@ -42,6 +42,21 @@ void pinmux_testutils_init(dif_pinmux_t *pinmux) {
   CHECK_DIF_OK(dif_pinmux_output_select(pinmux, kTopEarlgreyPinmuxMioOutIoc4,
                                         kTopEarlgreyPinmuxOutselUart0Tx));
 
+  // Enable pull-ups on UART0 TX/RX
+  dif_pinmux_pad_attr_t out_attr;
+  dif_pinmux_pad_attr_t in_attr = {
+      .slew_rate = 0,
+      .drive_strength = 0,
+      .flags = kDifPinmuxPadAttrPullResistorEnable |
+               kDifPinmuxPadAttrPullResistorUp};
+
+  CHECK_DIF_OK(dif_pinmux_pad_write_attrs(pinmux, kTopEarlgreyMuxedPadsIoc3,
```

Comment:
FYI: It looks like top_englishbreakfast is missing these pad constants.

Reply:
yeah this is super annoying.
I'm going to file an issue about the OTTF thing, but for now I'll probably need to lump it into the ifdef.